src/model/loss.py

```diff
@@ -23,11 +24,24 @@ def forward(self, hr_fake, hr_real, d_fake, d_real):
         """
 
         # Using straight MSE for now
-        g_loss = self.GLoss(hr_fake, hr_real, d_fake, content_choice='feat')
-        d_loss = self.DLoss(d_fake, d_real)
+        # g_loss = self.GLoss(hr_fake, hr_real, d_fake, content_choice='feat')
+        # d_loss = self.DLoss(d_fake, d_real)
+        g_loss = self.GLoss_new(hr_fake, hr_real, d_fake)
+        d_loss = self.DLoss_new(d_fake, d_real)
 
         return d_loss, g_loss
 
+    def GLoss_new(self, generated_image, target, d_fake):
```

Comment:
Did you have an issue with GLoss that lead you to define this? Or was it just a case of getting something working?

Reply:
I did not test your GLoss mse case. Unless torch.nn.MSELoss() and F.mse_loss() return very different result, they should be the same.